### PR TITLE
Bugs fixing #1

### PR DESCRIPTION
**Pull request by dtrungtin**

### CODE REVIEW

**Comment by dtrungtin** [comment]
Please merge the fix.

File: INPUT_SCHEMA.json [hunk 1]
```diff
@@ -65,7 +66,7 @@
       "editor": "hidden"
     },
     "checkInDate": {
-      "title": "Check in date",
+      "title": "Check in date (YYYY-MM-DD)",
```
Comment (pocesar) [inline INPUT_SCHEMA.json:70]: add the `pattern` here `\\d{4}-\\d{2}-\\d{2}`

File: readMe.md [hunk 2]
```diff
@@ -256,5 +256,3 @@ The actor produces output as data you can get by section.
  1. Visit the Place Page, e.g. `https://www.tripadvisor.cz/Hotel_Review-g186338-d1657415-Reviews-Park_Plaza_Westminster_Bridge_London-London_England.html`.
  2. The id is the number after `-d` in the url. For this url it would be `1657415`.
```
Comment (pocesar) [inline readMe.md:257]: merge the changes from master

File: src/tools/general.js [hunk 3]
```diff
@@ -64,23 +64,35 @@ async function resolveInBatches(promiseArray, batchLength = 10) {
 }
 
 const processReview = (review, remoteId) => {
-    const { text, title, rating, tripInfo, publishedDate, userProfile } = review;
+    const { text, title, rating, tripInfo, publishedDate, userProfile, photos } = review;
     const stayDate = tripInfo ? tripInfo.stayDate : null;
     let userLocation = null;
     let userContributions = null;
+    let userName = null;
+    let imageUrls = null;
```
Comment (pocesar) [inline src/tools/general.js]: should be `[]` so it's consistent to the output

File: src/tools/general.js [hunk 4]
```diff
@@ -64,23 +64,35 @@ async function resolveInBatches(promiseArray, batchLength = 10) {
...
         userContributions = sumReview;
         userLocation = hometown.fallbackString;
+        userName = username;
 
         if (hometown.location) {
             userLocation = hometown.location.additionalNames.long;
         }
     }
 
+    if (photos && photos.length > 0) {
```
Comment (pocesar) [inline src/tools/general.js:87]: adding photos should be an option on the INPUT_SCHEMA

**Review by pocesar — CHANGES_REQUESTED** [review]
@dtrungtin fix the conflict please

File: readMe.md (hunk 2 above)
Comment (dtrungtin) [reply readMe.md:257]: Both (master & my branch) are to remove the two last lines. It considers as nothing to merge from master.

File: INPUT_SCHEMA.json (hunk 1 above)
Comment (dtrungtin) [reply INPUT_SCHEMA.json:70]: Done

File: src/tools/general.js (hunk 3 above)
Comment (dtrungtin) [reply src/tools/general.js]: Done

File: src/tools/general.js (hunk 4 above)
Comment (dtrungtin) [reply src/tools/general.js:87]: Should open a new issue for new improvement. It's not scope of this bug.

**Comment by dtrungtin** [comment]
Please merge the changes.

File: INPUT_SCHEMA.json [hunk 5]
```diff
@@ -19,12 +20,13 @@
...
     "lastReviewDate": {
-      "title": "Reviews since date",
+      "title": "Reviews since date (YYYY-MM-DD)",
       "type": "string",
       "description": "It is a date where the scrapper stops loading older reviews ",
       "default": "2019-01-01",
       "prefill": "2019-01-01",
-      "editor": "textfield"
+      "editor": "textfield",
+      "patternValue": "\\d{4}-\\d{2}-\\d{2}"
```
Comment (pocesar) [inline INPUT_SCHEMA.json]: can't build, use `apify vis` for testing the schema. the property name is called `pattern` not `patternValue`

**Comment by pocesar** [comment]
I'm going to resolve the conflicts locally

**Comment by pocesar** [comment]
landed in 52aad13fe7b44f75d74d3c8634103fe942a97190

**Pull request closed without merging**